src/js-feature-toggle-manager.ts

```diff
@@ -33,7 +36,28 @@ export class FeatureToggleManager {
     return list;
   }
 
-  constructor() {
+  /**
+   * Provides a getter method for Feature Toggle Manager singleton object.
+   */
+  static get instance(): FeatureToggleManager {
+    if (undefined === this._instance) {
```

Comment:
In C# we could have a simple camelCase `instance` property, but that conflicts with the `instance()` getter method. I followed the example in the TypeScript documentation to use the `_instance`, but I'm open to other ideas for sure.